cache/store.go

```diff
@@ -177,6 +207,54 @@ func (ds *dataStore) removeDuplicateLayers(src, dst []*types.Layer) ([]*types.La
 	return layers, nil
 }
 
+func (ds *dataStore) UpdateManifestRef(namespace, ref string) error {
+	var res []byte
+	fn := func(txn *badger.Txn) error {
+		item, err := txn.Get([]byte(namespace))
+		if err != nil {
+			return err
+		}
+
+		return item.Value(func(v []byte) error {
+			res = make([]byte, len(v))
+			copy(res, v)
+			return nil
+		})
+	}
+
+	if err := ds.db.View(fn); err != nil {
+		return err
+	}
+
+	var md types.Metadata
+	err := json.Unmarshal(res, &md)
+	if err != nil {
+		return err
+	}
+	if len(md.Manifest.Config) == 0 {
```

Comment:
@guacamole i think we can remove this comparison. if the list is empty it should skip the loop anyway

Reply:
@jay-dee7 , removed the condition, ran tests they work just fine. please review and merge